openfisca_france_local/departements/eure_et_loir/caracteristiques_socio_demographiques.py

```diff
@@ -0,0 +1,22 @@
+from openfisca_france.model.base import Variable, Individu, MONTH
+
+class titre_sejour(Variable):
+    value_type = bool
+    label ="Bénéficiaire d'un titre de séjour"
+    entity = Individu
+    definition_period = MONTH
+    default_value = False
+
+class refugie(Variable):
+    value_type = bool
+    label = "La personne est réfugié"
```

Comment:
Reconnu au niveau administratif, non ? 🤔 
```suggestion
    label = "La personne dispose du statut de réfugié"
```